### DISCUSSION TITLE
FreshRSS (Linuxserver docker) + SWAG (Linuxserver docker) + Authelia, working for anyone?

### DISCUSSION THREAD

**Discussion by yieldhog** [FreshRSS/FreshRSS#6199] [Q&A]
I've been successfully using FreshRSS for awhile and its great, but, I wanted to tighten security a bit. I see there's a section in the docs regarding SSO/third party reverse proxies, however, it doesn't seem to address my setup, which is using Linuxserver docker containers for FreshRSS, SWAG, and Authelia. I am unable to figure out how to pass my Authelia username through to FreshRSS, as it only prompts me to enter my username/password. Has anyone figured this out and if so, any help appreciated.

**Comment by Alkarex** [comment]
Hello,
Our official Docker image is [`freshrss/freshrss`](https://hub.docker.com/r/freshrss/freshrss/) and the documentation assumes that.
I am not sure, but I do not believe that the LinuxServer image has OpenID at all.

* https://freshrss.github.io/FreshRSS/en/admins/09_AccessControl.html
* https://freshrss.github.io/FreshRSS/en/admins/16_OpenID-Connect.html
